telemetry-impls/summarize/send_trace.py

```diff
@@ -209,14 +209,9 @@ def process_job_blob(  # noqa: PLR0913
                 step_span.set_status(map_conclusion_to_status_code(step["conclusion"]))
                 step_span.end(end)
 
-                job_last_timestamp = max(end, job_last_timestamp)
-
-            job_end = max(
-                date_str_to_epoch(job["completed_at"], job_last_timestamp),
-                job_last_timestamp,
-            )
-            last_timestamp = max(job_end, last_timestamp)
-        job_span.end(job_end)
+            job_last_timestamp = max(end, job_last_timestamp)
```

Comment:
Can this line be moved up? I don't see any reason it needs to be below the `if:` block, and it would visually fit better closer to where `end` is defined above.

Reply:
yes, good call. That reads much better.